src/observable-hook.ts

```diff
@@ -32,6 +33,13 @@ export default <Store extends StoreObject>(key?: string, value?: Store): Store =
 	// get value for key
 	const keyValue = observableStore[resolvedKey];
 
+	// if we weren't passed in a key, this is a local obserable (not global),
+	const isLocalStore = !key;
+	if (isLocalStore) {
+		// if this is local, we should associate it with the element by putting it in the keyQueue
+		getKeyQueue(TRAM_KEY_QUEUE).push(resolvedKey);
+	}
```

Comment:
this is what adds this observable's key to the queue, which is later picked up and added to the element attributes.